COLLABORATOR_GUIDE.md

```diff
@@ -75,47 +77,86 @@ Collaborators or additional evidence that the issue has relevance, the
 issue may be closed. Remember that issues can always be re-opened if
 necessary.
 
+### Ready to land pull requests
+
+A pull request that is still awaiting the minimum review time is considered
+`ready` as soon as the CI got kicked off, it got at least one approval and
```

Comment:
Nit: Change first `got` to `has been` and the second `got` to `has`

Nit: Add a comma after `approval`